app/assets/javascripts/lookup_keys.js

```diff
@@ -69,7 +69,7 @@ function add_child_node(item) {
     var field   = '';
     if (assoc == 'lookup_keys') {
       $('#smart_vars .smart-var-tabs .active, #smart_vars .stacked-content .active').removeClass('active');
-      var pill = "<li class='active'><a data-toggle='pill' href='#new_" + new_id + "' id='pill_new_" + new_id + "'><div class='clip'>" + __('new') + "</div><span class='close pull-right'>&times;</span></a></li>"
+      var pill = "<li class='active'><a data-toggle='pill' href='#" + new_id + "' id='pill_" + new_id + "'><div class='clip'>" + __('new') + "</div><span class='close pull-right'>&times;</span></a></li>"
```

Comment:
it will create more issues than it will solve.

On Thu, Apr 9, 2015 at 12:08 PM, Ohad Levy <notifications@github.com> wrote:

> In app/assets/javascripts/lookup_keys.js
> <https://github.com/theforeman/foreman/pull/2055#discussion_r28046027>:
>
> > @@ -69,7 +69,7 @@ function add_child_node(item) {
> >      var field   = '';
> >      if (assoc == 'lookup_keys') {
> >        $('#smart_vars .smart-var-tabs .active, #smart_vars .stacked-content .active').removeClass('active');
> > -      var pill = "<li class='active'><a data-toggle='pill' href='#new_" + new_id + "' id='pill_new_" + new_id + "'><div class='clip'>" + __('new') + "</div><span class='close pull-right'>&times;</span></a></li>"
> > +      var pill = "<li class='active'><a data-toggle='pill' href='#" + new_id + "' id='pill_" + new_id + "'><div class='clip'>" + __('new') + "</div><span class='close pull-right'>&times;</span></a></li>"
>
> maybe all of the strong params needs to be merged separately from thsi
> branch? @shlomizadok <https://github.com/shlomizadok> @eLobato
> <https://github.com/eLobato> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/2055/files#r28046027>.
>